pytest/tests/mocknet/mirror.py

```diff
@@ -576,25 +607,14 @@ def run_env_cmd(args, traffic_generator, nodes):
         'update-binaries',
         help=
         'Update the neard binaries by re-downloading them. The same urls are used.'
-    )
+        + ' Add or override the URLs with --neard-binary-url.')
+    update_binaries_parser.add_argument('--neard-binary-url', type=str)
+    update_binaries_parser.add_argument('--update-epoch-height', type=int)
```

Comment:
can you add help comments for these? especially the effect of update-epoch-height